src/EditorFeatures/Core/Tagging/TaggerContext.cs

```diff
@@ -68,7 +65,6 @@ internal class TaggerContext<TTag> where TTag : ITag
             this.CaretPosition = caretPosition;
             this.TextChangeRange = textChangeRange;
 
-            _spansTagged = spansToTag.SelectAsArray(ds => ds.SnapshotSpan);
```

Comment:
by default, if the tagger didn't change aythnig, you just got back the original set you were asked to tag.  that logic is preserved below.